draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -841,6 +867,25 @@ the SD-JWT.
 
 # Privacy Considerations {#privacy_considerations}
 
+## Confidentiality during Transport
+
+If the SD-JWT and associated Disclosures are transmitted over an insecure
+channel during issuance or presentation, an adversary may be able to
+intercept and read the End-User's personal data or correlate the information with previous uses of the same SD-JWT.
+
+Usually, transport protocols for issuance and presentation of credentials
+are designed to protect the confidentiality of the transmitted data, for
+example, by requiring the use of TLS.
+
+This specification therefore considers the confidentiality of the data to be
+provided by the transport protocol and does not specify any encryption
+mechanism.
+
+Implementers MUST ensure that the transport protocol provides confidentiality
+if the privacy of End-User data or correlation attacks are a concern. Implementers MAY define a
+JWT envelope format (as described in (#enveloping)) to encrypt the SD-JWT
+and associated Disclosures when transmitted over an insecure channel.
```

Comment:
```suggestion
Implementers MUST ensure that the transport protocol provides confidentiality,
if the privacy of End-User data or correlation attacks are a concern. Implementers MAY define an
envelope format (such as described in (#enveloping) or nesting the SD-JWT Combined Format as
the plaintext of a JWE) to encrypt the SD-JWT
and associated Disclosures when transmitted over an insecure channel
```

Reply:
Should 'plaintext' be 'payload'? Other than that, looks good!

Reply:
as mentioned on one of the messaging platforms - I think nesting the SD-JWT and Disclosures directly in a JWE is an appropriate approach in some/many cases. This suggestion just tries to leave room for that.

Reply:
> Should 'plaintext' be 'payload'? Other than that, looks good!

AFAIK JWE/T uses 'plaintext' and 'payload' somewhat interchangeably and sometimes together.  I updated the suggestion to have both "plaintext payload".

Reply:
I see it says 'plaintext payload' now. Thanks, I'll merge this!